lib/core/crypto/dart_esr/src/encoding_options.dart

```diff
@@ -14,8 +13,13 @@ SigningRequestEncodingOptions defaultSigningRequestEncodingOptions(
       textEncoder: DefaultTextEncoder(),
       textDecoder: DefaultTextDecoder(),
       zlib: DefaultZlibProvider(),
-      // TODO(n13): need to pass nodeUrl parameter into EOSClient here -
-      abiProvider: DefaultAbiProvider(GetIt.I.get<EOSClient>()),
```

Comment:
@n13  this change is important. I replace the injection of EOSClient in favor of creating them each time